### ISSUE TITLE
`claude_batch` outranks `audit` in Phase 6 — model output overwrites the owner's own labels

### ISSUE THREAD

**Issue by BFox010** [BFox010/Music-Enrichment#67]
## Problem

Two modules encode opposite trust orders for mood sources, and the wrong one wins where it matters.

`classify_moods.py` checks the Claude-review index **before** the audit index, so model output overwrites the owner's hand labels:

```python
# Priority 1: Claude review (highest confidence)
if key in claude_index:
    track["mood_source"] = "claude_batch"
    ...
    continue

# Priority 2: direct audit hit (also high confidence — owner-labeled)
if key in audit_index:
    track["mood_source"] = "audit"
```

`pipeline/classify_moods.py:585` and `:594`.

`resolve_identity.py:59` encodes the opposite:

```python
"audit": 4, "claude_batch": 3, "centroid": 2, None: 0,
```

CLAUDE.md agrees with `resolve_identity`, not with Phase 6:

> **`mood_source: "audit"` is the owner's own labelling** — the training signal the whole classifier is built on. A fresher centroid pass must never overwrite one.

`classify_moods.py:510` even states the same principle in a comment while building training data — "``audit`` counts as ground truth — claude_batch rows are themselves model output" — and then the application loop below it does the reverse.

## Impact

11 tracks on disk currently carry `claude_batch` labels on rows the owner labelled by hand in `mood_audit.csv`. Small in count, but it is the highest-value data in the repo being overwritten by a model, and the count grows with every Claude batch run.

## Proposed change

Swap priorities 1 and 2 in `classify_moods`'s application loop, and make `_SOURCE_RANK` the single definition of trust order so the two modules cannot drift apart again.

Worth checking whether `OWNER_LABEL_SOURCES` (`classify_moods.py:400`, which lumps `audit` and `claude_batch` together as owner labels) should also be narrowed — a Claude batch row is not an owner label, and treating it as one is what makes this easy to get wrong.

## Acceptance criteria

- [ ] `audit` beats `claude_batch` in Phase 6's application loop
- [ ] Trust order defined once and shared, not duplicated between `classify_moods` and `resolve_identity`
- [ ] Test asserting an audit label survives a `claude_batch` hit on the same track

Background: pipeline audit §1.4, `docs/pipeline_audit_2026-08-23.md` on branch `claude/pipeline-audit-quality-bxbdv0` (not merged to `main`).
